MIGRATION.md

```diff
@@ -537,3 +556,652 @@ import { Grid, GridItem } from '@contentful/f36-components';
+
+```jsx static=true
+import { RadioButton } from '@contentful/forma-36-react-components';
+
+<RadioButton name="someOption" id="radio" labelText="some label" />;
+```
+
+_Note: This example would render only the input and the `labelText` would be set as the `aria-label`._
+
+You will do the following:
+
+```jsx static=true
+import { Radio } from '@contentful/f36-components';
+
+// This would render the exact same as on version 3
+<Radio name="someOption" id="radio" aria-label="some label" />
+
+// You can also pass the label and help text to the radio
+<Radio name="someOption" id="radio" helpText="Some help text">
+  This label will be rendered at the side of the input
+</Radio>
+```
+
+We also introduced API changes for the radio. This is an overview of the renamed properties:
+
+- `required` was renamed to `isRequired`;
+- `labelText` should be replaced by `aria-label` when not passing a label to the input;
+- `checked` was renamed to `isChecked`;
+- `disabled` was renamed to `isDisabled`.
+
+#### How to migrate your RadioButton components
+
+Run the [codemod](https://github.com/contentful/forma-36/tree/forma-v4/packages/forma-36-codemod) to migrate your v3 `RadioButton` component to the new version:
+
+`npx @contentful/f36-codemod`
+
+If you decide to do it manually, you would need to transform your code:
+
+```jsx static=true
+import { RadioButton } from '@contentful/forma-36-react-components';
+
+<RadioButton
+  id="radio"
+  disabled={false}
+  checked={true}
+  labelText="some label"
+/>;
+```
+
+into this new version:
+
+```jsx static=true
+import { Radio } from '@contentful/f36-components';
+
+<Radio
+  id="radio"
+  isDisabled={false}
+  isChecked={true}
+  aria-label="some label"
+/>;
+```
+
+### Select
+
+`Select` component becomes compound component in version 4. You can use `Select` and `Select.Option` , with just importing `Select`. You would need to transform your existing code:
+
+```tsx static=true
+<Select id="optionSelect" name="optionSelect" width="large">
+  <Option value="optionOne">Option 1</Option>
+  <Option value="optionTwo">Long Option 2</Option>
+</Select>
+```
+
+into the following:
+
+```tsx static=true
+<Select id="optionSelect" name="optionSelect">
+  <Select.Option value="optionOne">Option 1</Select.Option>
+  <Select.Option value="optionTwo">Long Option 2</Select.Option>
+</Select>
+```
+
+We also introduced some API changes for the select, check the props that had changes:
+
+- `width` was removed;
+- `required` was renamed to `isRequired`;
+- `hasError` was renamed to `isInvalid`;
+- `disabled` was renamed to `isDisabled`;
+- The `Select.Option` component can receive a `isDisabled` prop instead of the `disabled`.
+
+#### How to migrate your Select components
+
+Run the [codemod](https://github.com/contentful/forma-36/tree/forma-v4/packages/forma-36-codemod) to migrate your v3 `Select` component to the new version:
```

Comment:
To migrate your v3 `Select` component, run the following [codemod](https://github.com/contentful/forma-36/tree/forma-v4/packages/forma-36-codemod):